linker-interface/shared/src/main/scala/org/scalajs/linker/interface/Semantics.scala

```diff
@@ -75,13 +75,23 @@ final class Semantics private (
     finalizeHash(acc, 6)
   }
 
+  def serialized: String =
```

Comment:
I would not expose these, but just make them private to `interface` (maybe also on the companion).

Reply:
Done :+1: